src/com/oltpbenchmark/benchmarks/tpcc/procedures/NewOrder.java

```diff
@@ -303,11 +304,17 @@ private void newOrderTransaction(int w_id, int d_id, int c_id,
 
         stockQuantities[ol_number - 1] = s_quantity;
 
+        if (ol_i_id == 1) {
```

Comment:
Do we need this log here or was it just for debugging?

Reply:
Removed